lib/generators/boring/rswag/install/install_generator.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+module Boring
+  module Rswag
+    class InstallGenerator < Rails::Generators::Base
+      DEFAULT_RAILS_PORT = '3000'
+
+      desc "Adds rswag gem to the application"
+
+      class_option :rails_port,
+                   type: :string,
+                   desc: "Tell us the port number where you normally run your rails app. Defaults to PORT #{DEFAULT_RAILS_PORT}",
+                   default: DEFAULT_RAILS_PORT
+
+      def verify_presence_of_rspec_gem
+        gem_file_content_array = File.readlines("Gemfile")
+
+        rspec_is_installed = gem_file_content_array.any? { |line| line.include?("rspec-rails") }
+
+        return if rspec_is_installed
+
+        say "We couldn't find rspec-rails gem which is a dependency for the rswag gem. Please configure rspec and run the generator again!"
+
+        abort
+      end
+
+      def add_rswag_gem
+        say "Adding rswag gem", :green
+
+        run "bundle add rswag"
+      end
+
+      def run_rswag_install_generator
+        say "\nRunning rswag install generator to add required files", :green
+
+        run "bundle exec rails generate rswag:install"
+      end
+
+      def update_api_host
+        say "\nUpdating API Host URL"
+
+        rails_port = options[:rails_port]
+
+        gsub_file "spec/swagger_helper.rb",
+                  "url: 'https://{defaultHost}'",
+                  "url: 'http://{defaultHost}'"
+        gsub_file "spec/swagger_helper.rb",
+                  "default: 'www.example.com'",
+                  "default: 'localhost:#{rails_port}'"
+      end
+
+      def show_whats_next_message
```

Comment:
can you move this to readme.md file? Each generator can have a readme that explains next steps after install the generator. You can find examples in the codebase for same.